src/runtime/polyfill/process.ts

```diff
@@ -1,6 +1,9 @@
-import _process from "../node/process/index";
-import _global from "./global-this";
+import type nodeProcess from "node:process";
+import unenvProcess from "../node/process/index";
 
-_global.process = _global.process || _process;
-
-export default _global.process;
+// TODO: or should we Object.assign into globalThis.process to preserve the identity
```

Comment:
@pi0 I'd love your take on this one. I'm torn as both options have pros and cons

Reply:
I would go with `Object.assign` too as it preserves constructor 👍🏼

Reply:
ok, sgtm. I'll update, but I think we should file an issue to do a pass through the library to ensure that we follow this consistently everywhere.